src/engine/UNSATCertificate.h

```diff
@@ -0,0 +1,282 @@
+/*********************                                                        */
+/*! \file UNSATCertificate.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Omri Isac, Guy Katz
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2022 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** [[ Add lengthier description here ]]
+ **/
+
+
+#ifndef __UNSATCertificate_h__
```

Comment:
UNSAT --> Unsat
(filename too, but do that later, so that it doesn't erase the history of comments)